src/components/JdsDataTable/DataTable.stories.js

```diff
@@ -75,8 +75,11 @@ hideArgTypes(EmptyState, [
   'headers',
   'items',
   'localSort',
+  'showSelect',
+  'itemKey',
   'loading'
 ])
 hideEvents(EmptyState, [
-  'change:sort'
+  'change:sort',
+  'change:select'
 ])
```

Comment:
Better you do enter it

Reply:
thank you @adzharamrullah , fixed [here](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/90/commits/e3e2e21bc98cee4442fa0869018e7537f7bcb322)